docker/Dockerfile.helm-operator

```diff
@@ -1,8 +1,36 @@
-FROM alpine:3.9
+FROM debian:stable-slim
 
 WORKDIR /home/flux
 
-RUN apk add --no-cache openssh ca-certificates tini 'git>=2.12.0'
+RUN echo "deb http://deb.debian.org/debian stretch-backports main" | tee -a /etc/apt/sources.list.d/stretch-backports.list && \
+  apt-get update && apt-get install -y --no-install-recommends \
+    openssh-client \
+    ca-certificates && \
+  apt-get install -t stretch-backports -y --no-install-recommends git && \
+  rm -rf /var/lib/apt/lists/*
+
+ENV TINI_VERSION 0.18.0
+ENV TINI_GPG_KEY 595E85A6B1B4779EA4DAAEC70B588DFF0527A9B7
+ENV GPG_KEY_SERVERS_LIST ha.pool.sks-keyservers.net \
+                         hkp://p80.pool.sks-keyservers.net:80 \
+                         keyserver.ubuntu.com \
+                         hkp://keyserver.ubuntu.com:80 \
+                         pgp.mit.edu
+
+RUN cd /tmp && \
+  apt-get update && apt-get install -y curl gnupg && \
+  export GNUPGHOME="$(mktemp -d)" && \
+  for server in $(shuf -e $GPG_KEY_SERVERS_LIST); do \
+    gpg --keyserver "$server" --recv-keys $TINI_GPG_KEY && break || : ; \
+  done && \
+  gpg --fingerprint $TINI_GPG_KEY | grep -q "6380 DC42 8747 F6C3 93FE  ACA5 9A84 159D 7001 A4E5" && \
+  curl -sSL "https://github.com/krallin/tini/releases/download/v${TINI_VERSION}/tini_${TINI_VERSION}.deb" -o tini.deb && \
+  curl -sSL "https://github.com/krallin/tini/releases/download/v${TINI_VERSION}/tini_${TINI_VERSION}.deb.asc" -o tini.deb.asc && \
+  gpg --verify tini.deb.asc tini.deb && \
+  apt-get install /tmp/tini.deb && \
+  apt-get purge --auto-remove -y curl gnupg && \
+  rm -rf "$GNUPGHOME" tini.deb* && \
+  rm -rf /var/lib/apt/lists/*
```

Comment:
This is a looot of code for just installing tini. Have you tried the package from testing/unstable https://packages.debian.org/search?keywords=tini ?